library/models/library.py

```diff
@@ -437,11 +437,12 @@ def issue_book(self):
         @return : True
         '''
 
-        curr_dt = datetime.now()
+        curr_dt = fields.Date.today()
+
```

Comment:
@dhruvitaSerpentCS Can you comment here within Pr what was the issue and why the new code ?
Thanks.

Reply:
Earlier in old code we get datetime but we required only date field.
In old code we get the traceback error.